packages/ui/src/lib/layouts/BasePage.svelte

```diff
@@ -0,0 +1,105 @@
+<script lang="ts">
+import { createEventDispatcher } from 'svelte';
+
+import { LinearProgress } from '..';
+import CloseButton from '../button/CloseButton.svelte';
+import Link from '../link/Link.svelte';
+
+export let title: string;
+export let titleDetail: string | undefined = undefined;
+export let subtitle: string | undefined = undefined;
+export let breadcrumbLeftPart: string | undefined = undefined;
+export let breadcrumbRightPart: string | undefined = undefined;
+export let breadcrumbTitle: string | undefined = '';
```

Comment:
Not related to this PR but I know that `left | right` breadcrumb has been used in other component, but I really don't like it, as it limit to only two when we could need more ?

Reply:
I agree, but this PR is just merging two copies of that breadcrumb code into one parent, not trying to improve it at this point.